src/ext/oblivc/obliv_bits.h

```diff
@@ -40,13 +43,17 @@ void __obliv_c__setBitOr(OblivBit* dest,const OblivBit* a,const OblivBit* b);
 void __obliv_c__setBitXor(OblivBit* dest,const OblivBit* a,const OblivBit* b);
 void __obliv_c__setBitNot(OblivBit* dest,const OblivBit* a);
 void __obliv_c__flipBit(OblivBit* dest); // Avoids a struct copy
+void __obliv_c__floatAdd(OblivBit* dest,const OblivBit* a,const OblivBit* b);
```

Comment:
Yes, while those three all do work over more than one bit, the naming conventions for the second and third are actually copied from their int-addressing counterparts so that they fit in with the rest of the code.